dbt/adapters/redshift/connections.py

```diff
@@ -233,27 +233,26 @@ def connect():
 class RedshiftConnectionManager(SQLConnectionManager):
     TYPE = "redshift"
 
-    def _get_backend_pid(self):
-        sql = "select pg_backend_pid()"
-        _, cursor = self.add_query(sql)
-
-        res = cursor.fetchone()
-        return res[0]
-
     def cancel(self, connection: Connection):
+        pid = connection.backend_pid
```

Comment:
```suggestion
        pid = connection.backend_pid # type: ignore
```

Reply:
Apparently `black` required two spaces before the #.